dlib_face_detection/include/dlib_face_detection/detection_viewer.h

```diff
@@ -0,0 +1,51 @@
+#ifndef __DETECTION_VIEWER_H__
+#define __DETECTION_VIEWER_H__
+
+
+#include <ros/ros.h>
+
+
+// ROS message
+#include <sensor_msgs/Image.h>
+#include <sensor_msgs/image_encodings.h>
+#include <cob_perception_msgs/DetectionArray.h>
+
+#include <iostream>
+#include <string.h>
+
+// message_filter
+#include <message_filters/subscriber.h>
+#include <message_filters/synchronizer.h>
+#include <message_filters/sync_policies/approximate_time.h>
+
+// opencv
+#include <opencv/cv.h>
+#include <opencv/ml.h>
+#include <cv_bridge/cv_bridge.h>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv/highgui.h>
```

Comment:
I haven't detected any problems with that and the Travis build was clean too. So i think there are no conflicts.